01. ML Challenges/homework_1.ipynb

```diff
@@ -1,936 +1,1344 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        res = ((billy &gt; 0.5)&amp;(herrington &gt; 0.5)).nonzero()[0]
</pre><p>I'll show you who is the boss of this gym</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kryzhikov/Sample-ML-Repo/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>